document-generator-accounts/src/test/java/uk/gov/companieshouse/document/generator/accounts/mapping/smallfull/mappers/SmallFullIXBRLMapperTest.java

```diff
@@ -0,0 +1,311 @@
+        assertEquals(new Long(VALUE_ONE), fixedAssets.getTangibleAssets().getCurrentAmount());
+        assertEquals(new Long(VALUE_TWO), fixedAssets.getTotalFixedAssetsCurrent());
+
+        if (isMultiYearFiling == true) {
+
+            assertEquals(new Long(VALUE_ONE), fixedAssets.getTangibleAssets().getPreviousAmount());
+            assertEquals(new Long(VALUE_TWO), fixedAssets.getTotalFixedAssetsPrevious());
+        }
+    }
+
+    private void assertCurrentAssets(CurrentAssets currentAssets, boolean isMultiYearFiling) {
+
+        assertNotNull(currentAssets);
+        assertEquals(new Long(VALUE_ONE), currentAssets.getCashAtBankAndInHand().getCurrentAmount());
+        assertEquals(new Long(VALUE_TWO), currentAssets.getDebtors().getCurrentAmount());
+        assertEquals(new Long(VALUE_THREE), currentAssets.getStocks().getCurrentAmount());
+        assertEquals(new Long(VALUE_ONE), currentAssets.getCurrentTotal());
+
+        if (isMultiYearFiling == true) {
+
+            assertEquals(new Long(VALUE_ONE), currentAssets.getCashAtBankAndInHand().getPreviousAmount());
+            assertEquals(new Long(VALUE_TWO), currentAssets.getDebtors().getPreviousAmount());
+            assertEquals(new Long(VALUE_THREE), currentAssets.getStocks().getPreviousAmount());
+            assertEquals(new Long(VALUE_ONE), currentAssets.getPreviousTotal());
+        }
+    }
+
+    private void assertCapitalAndReserve(CapitalAndReserve capitalAndReserve, boolean isMultiYearFiling) {
+
+        assertNotNull(capitalAndReserve);
+        assertEquals(new Long(VALUE_ONE), capitalAndReserve.getCalledUpShareCapital().getCurrentAmount());
+        assertEquals(new Long(VALUE_TWO), capitalAndReserve.getOtherReserves().getCurrentAmount());
+        assertEquals(new Long(VALUE_THREE), capitalAndReserve.getProfitAndLoss().getCurrentAmount());
+        assertEquals(new Long(VALUE_ONE), capitalAndReserve.getSharePremiumAccount().getCurrentAmount());
+        assertEquals(new Long(VALUE_TWO), capitalAndReserve.getTotalShareHoldersFund().getCurrentAmount());
+
+        if (isMultiYearFiling == true) {
+
+            assertEquals(new Long(VALUE_ONE),capitalAndReserve.getCalledUpShareCapital().getPreviousAmount());
+            assertEquals(new Long(VALUE_TWO), capitalAndReserve.getOtherReserves().getPreviousAmount());
+            assertEquals(new Long(VALUE_THREE), capitalAndReserve.getProfitAndLoss().getPreviousAmount());
+            assertEquals(new Long(VALUE_ONE), capitalAndReserve.getSharePremiumAccount().getPreviousAmount());
+            assertEquals(new Long(VALUE_TWO), capitalAndReserve.getTotalShareHoldersFund().getPreviousAmount());
+        }
+    }
+
+    private void assertCompanyProfileMapped(Company company) {
+
+        assertNotNull(company);
+        assertEquals("companyName", company.getCompanyName());
+        assertEquals("companyNumber", company.getCompanyNumber());
+        assertEquals("jurisdiction", company.getJurisdiction());
+    }
```

Comment:
I would create constants for these names: 
`companyName`
`companyNumber`
`jurisdiction`

Reply:
done